### DISCUSSION TITLE
Application is killed on startup in Ubuntu

### DISCUSSION THREAD

**Discussion by ognenbek** [LGouellec/streamiz#209] [Q&A] [answered]
I am trying to run streamiz on Ubuntu but it fails when it tries to create the new KafkaStream. Im trying to do a Kstream Kstream join with rocksDb store, however i got the same issue even with inMemoryStore. Tried the application locally on windows machine and did not have any issue.
Checked the logs and the application is killed without any exception thrown, also noticed that before the application is killed the cpu is increasing almost to 100%.

The error in the log is the following:
![log](https://user-images.githubusercontent.com/18530194/204032231-4503dfc2-701d-412f-8ea3-76c9715c4268.PNG)

The configuration:
![config](https://user-images.githubusercontent.com/18530194/204032306-34f53113-36d0-4b69-8afb-b6dad50647d2.PNG)

the failing point in the code:
![failing_point](https://user-images.githubusercontent.com/18530194/204032586-a14aa490-2067-4436-91f9-85a533ce251f.PNG)

Installed the additional packages:
sudo apt install -y libc6-dev libgflags-dev libsnappy-dev liblz4-dev libzstd-dev
sudo apt install -y bzip2 lz4 librocksdb-dev

Ubuntu version: Ubuntu 20.04.3 LTS

Please let me know If somebody already had this issue and a possible fix.

**Comment by LGouellec** [comment]
Could you upgrade to 1.4.0-RC5 and enable debugs logs : 

``` csharp
config.Debug = "security,broker,protocol";
config.Logger = LoggerFactory.Create(builder => { 
builder.SetMinimumLevel(LogLevel.Debug);
builder.AddConsole(); // add your logger : console, log4net, whatever
});

```

**Reply by ognenbek** [reply]
I am already using the latest version of streamiz. Tried the suggested configurations but still have the same log. Are there any other suggestions I can try?

**Reply by LGouellec** [reply]
The only thing : checkout the code and run your app in debug with streamiz to see if any exception is thrown ?

**Reply by ognenbek** [reply]
Hey, we ran the application in debug mode but no exception was thrown. also added try catch round the new `KafkaStream(topology, config); `with catching `Win32Exception` and `Exception` but still we are getting the same error.

![image](https://user-images.githubusercontent.com/18530194/204090599-886f6eb8-c79b-4df1-bf75-50702b6aade1.png)

**Reply by LGouellec** [reply]
Did you try to checkout the streamiz code and run with the debugger attached ?

**Reply by ognenbek** [reply]
It turned out that the bootstrat.servers property was misconfigured. We used an HTTP://URL which was causing an issue, adding only the URL solved the issue. Thank you for the support @LGouellec :)

**Comment by LGouellec** [comment, marked as answer]
You are welcome ! I mark this discussion as "Answered"
